dev-system/dev/user.clj

```diff
@@ -130,6 +130,7 @@
   (if bool
     (q-config/set-queue-type! "aws")
     (q-config/set-queue-type! "local")))
+    ;(q-config/set-queue-type! "memory"))) ; why is this not used over local?
```

Comment:
Remove this line.

Reply:
so I have a question about this, and I did not want to throw the code away. I forget who, but one of the people I worked with mentioned that "local" is incorrect and that we should be using "memory". So I left it here to start the conversation. Why is local used and not memory?

Reply:
The "local" is obsolete. You should change it to "memory" or not set it at all because it doesn't appear that the value is being used. but not leave this commented line here.